src/main/java/com/coyoapp/tinytask/web/TaskController.java

```diff
@@ -27,9 +24,9 @@
   private final TaskService taskService;
 
   @PostMapping
```

Comment:
javax.validation.constraints for @Valid no longer works due to usage of @RequestParam

Reply:
Roger that! missed that.